website/docs/how-to/how-to-add-sso-azure-saml.md

```diff
@@ -0,0 +1,117 @@
+
+**b) In the Azure AD gallery, select the option to create your own application.**
+
+![The Azure AD gallery. The "create your own application" button is highlighted.](/img/sso-azure-saml-create-own-app.png)
+
+**c) Next, provide the application with a name. When asked what you're looking to do with the application, select the "Integrate any other application you don't find in the gallery (Non-gallery)" option.**
+
+![Azure application creation form. The name is set to "UnleashSSO" and the "non-gallery" option is selected.](/img/sso-azure-saml-name-app.png)
+
+### Step 2: Configure SSO via SAML in Azure AD {#step-2}
+
+**a) On the single sign-on page ("single sign-on" in the side bar), select the "SAML" option**
+
+![Azure: SSO method selection page](/img/sso-azure-saml-saml-choice.png)
+
+**b) Section 1: Basic SAML Configuration {#basic-saml-configuration}**
+
+When configuring SSO with SAML, you'll need to add an **identifier** and a **reply URL**.
+The **identifier** is your Unleash URL. (For hosted instances, that's usually `https://<region>.app.unleash-hosted.com/<instanceName>`).
+
+The **reply URL** is the Unleash callback URL. The Unleash callback URL is available on the Unleash SSO configuration page, and is typically your Unleash URL followed by `/auth/saml/callback`.
+
+![Azure: The basic SAML configuration form with example values filled out for the required fields.](/img/sso-azure-saml-section-one.png)
+
+**c) Section 2: Attributes & Claims {#attributes-and-claims}**
+
+1. Set the "name identifier format" to "Email address".
+2. Select "attribute" as the source.
+3. Enter "user.mail" in the source attribute field.
+
+Optionally, you can also provide a first name and a last name. If provided, these will be used to enrich the data in Unleash.
+
+![Azure: The manage claim form with email configuration filled out](/img/sso-azure-saml-unique-id-email-id.png)
+![Azure: The list of claims used by the SAML setup, including the optional claims for given name and surname](/img/sso-azure-saml-attributes-claim.png)
+
+> Please make sure to replace URLs with the public URL for your Unleash instance. This will require correct region prefix and the instance name. 
+>
+> The correct format is: https://**[region]**.app.unleash-hosted.com/**[instanceName]**/auth/saml/callback
+
+**d) Sections 3 and 4: Azure AD setup details {#azure-details}**
+
+You will need some details from section 3 and 4 of the SAML setup form to configure the integration within Unleash. These details are:
+- Azure AD Identifier (from section 4)
+- Login URL (from section 4)
+- X.509 Certificate (in the Federation Metadata XML from section 3)
+
+![Azure: Section 3 contains a download link for the Federation Metadata XML file. Section 4 lists the Azure AD identifier and the login URL](/img/sso-azure-saml-azure-details.png)
+![Azure: Within the Federation Metadata XML file, find the `X509Certificate` tag. You'll need the content within that tag.](/img/sso-azure-saml-x509cert.png)
```

Comment:
These both feel like they should be figures instead of images. I think the alt text would be a useful caption for anyone. what do you think?